deploy/cloudbuild-release.yaml

```diff
@@ -1,110 +1,67 @@
-# This cloudbuild is run on the creation of new tags, which should signify releases.
 timeout: 1800s
 
 steps:
+# Set up builder for muti-arch builds.
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['run', '--privileged', 'linuxkit/binfmt:v0.7']
+  id: 'initialize-qemu'
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['buildx', 'create', '--name', 'mybuilder']
```

Comment:
Is `buildx` meant to be `build`?

Reply:
no buildx is how you invoke the buildkit stuff.

Reply:
no its `buildx` to create a cross platform builder and then use it to later.